changelog.txt

```diff
@@ -1,8 +1,25 @@
+Version 2.0.6
+- Use fixed thread pool for silent requests
```

Comment:
Minor formatting nit: other sections have a `------` after the version line

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/886/commits/8636723d380562b9e8afc37c5f41be751807d9ee